tracim/tracim/tests/functional/test_calendar.py

```diff
@@ -0,0 +1,220 @@
+import time
+
+import caldav
+import transaction
+from caldav.lib.error import AuthorizationError
+from nose.tools import eq_, ok_
+import requests
+from requests.exceptions import ConnectionError
+from sqlalchemy.orm.exc import NoResultFound
+
+from tracim.config.app_cfg import daemons
+from tracim.lib.workspace import WorkspaceApi
+from tracim.model import DBSession
+from tracim.tests import TestCalendar as BaseTestCalendar
+from tracim.model.auth import User
+from tracim.model.data import Content
+
+
+class TestCalendar(BaseTestCalendar):
+    def setUp(self):
+        super().setUp()
+        time.sleep(3)  # TODO - 20160606 - Bastien: sleep to wait ...
+        # ... radicale daemon started. We should lock something somewhere !
+
+    def test_func__radicale_connectivity__ok__nominal_case(self):
+        radicale_base_url = self._get_base_url()
+
+        try:
+            response = requests.get(radicale_base_url)
+            eq_(response.status_code, 401, 'Radicale http response is 401')
```

Comment:
'Radicale http response is 401' => this is the message shown when the test fail (so the HTTP response is NOT 401... please change the wording + show the status_code returned).

This message should help understand why the test fail